pkg/listener/listener.go

```diff
@@ -73,27 +76,24 @@ func (l *Listener) DisplayHelloMessage() {
 }
 
 func (l *Listener) Name() string {
-	hostname, err := os.Hostname()
-	if err != nil {
-		panic(err)
-	}
+	randBytes := make([]byte, 20)
 
-	randBytes := make([]byte, 10)
-
-	_, err = rand.Read(randBytes)
+	_, err := rand.Read(randBytes)
 	if err != nil {
 		panic(err)
 	}
 
-	randSuffix := fmt.Sprintf("%x", randBytes)
-
-	return "sh-" + hostname + "-" + randSuffix
+	return fmt.Sprintf("%x", randBytes)
```

Comment:
This will change the name of the agnets. Previously they were `sh-{hostname}-{random-suffix}`, but on many places it turned out ugly. For example on AWS it was `sh-ip-172-9-1-10-31312sdfa`. 

I dropped everything on the front, and made the random part longer. Now it is only `31312sdfa231jkj312`.